slam-ws/transform/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(transform)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
```

Comment:
Why do we have these dependencies? In theory we don't need any of them since there is no code, but we definitely don't need both roscpp and rospy. Seems cleaner to remove all the dependencies